config/config.go

```diff
@@ -1,74 +1,35 @@
 package config
 
 import (
-	"os"
-
-	"github.com/cloudfoundry-community/go-cfenv"
+	"github.com/kelseyhightower/envconfig"
 
 	"github.com/jinzhu/gorm"
 	_ "github.com/jinzhu/gorm/dialects/postgres"
 	_ "github.com/lib/pq"
 )
 
-func Connect() (*gorm.DB, error) {
-	return gorm.Open("postgres", os.Getenv("DATABASE_URL"))
-}
-
 type Settings struct {
-	Port         string
-	BrokerUser   string
-	BrokerPass   string
-	DatabaseUrl  string
-	Email        string
-	AcmeUrl      string
-	Bucket       string
-	AwsAccessKey string
-	AwsSecretKey string
-	AwsRegion    string
+	Port               string `envconfig:"port";default:"3000"`
+	Username           string `envconfig:"username";required:"true"`
+	Password           string `envconfig:"password";required:"true"`
```

Comment:
Hmm, since we're connecting to multiple services, being more specific with what these credentials are for would be helpful.